pyproject.toml

```diff
@@ -68,6 +68,7 @@ openpyxl = "^3.0.9"
 Flask-Cors = "^3.0.10"
 pdoc = "^12.2.0"
 dateparser = "^1.1.4"
+cryptography = "38.0.4"
```

Comment:
Do other minor releases of version `38.x.x` work or only this specific one? Could we open it up to `^38.0.4` or would that break things?

Reply:
Not sure! My guess is anything before 38.0.4 would work.

Here's another possible [fix](https://github.com/pyca/cryptography/issues/7959)

Reply:
I noticed that possibility in the SO thread too. It might be better to try leaving `cryptography` as the latest major version and adding `pyopenssl` to our .toml with the indicated version and above. I noticed in our .lock for the `pyopenssl` extra its `>=20.0.0` so that could be where the issue is coming from.